### ISSUE TITLE
Tuya Third-patry authorization management

### ISSUE THREAD

**Issue by Xenomes** [jasonacox/tinytuya#465]
Hi jasonacox,
I noticed that Home Assistant has a 'Third-party authorization management' feature with a user code that is provided in the Tuya-based app. Do you think this could be possible for your project to bypass the 6-month renewal?
![afbeelding](https://github.com/jasonacox/tinytuya/assets/11648297/6b16936f-6c75-4451-880f-e2b175e3c563)

**Comment by jasonacox** [comment]
Is that for cloud access? I suspect it is and wouldn't help with those wanting local access. But may be worth exploring.

What is the HA plugin called?

**Comment by Xenomes** [comment]
Yes is for cloud access at HA, If users add a new device after 6 months of the last dump of the json they need to extend the subscription. Here is the location https://github.com/home-assistant/core/tree/dev/homeassistant/components/tuya

**Comment by Xenomes** [comment]
I hope they answer, but I am afraid not.

**Comment by jasonacox** [comment]
Thank you for reporting this issue! This issue has been open for over 2 years with no recent activity. We are closing it to help keep the issue tracker focused and manageable.

If this problem is still present in the latest version of TinyTuya, please feel free to reopen this issue or open a new one with updated details (TinyTuya version, Python version, device type, and steps to reproduce). We'd love to help!

Thank you for your contribution to TinyTuya! 🙏
